backend/src/API/shoutoutAPI.ts

```diff
@@ -42,3 +42,17 @@ export const hideShoutout = async (
   if (!shoutout) throw new NotFoundError(`Shoutout with uuid: ${uuid} does not exist!`);
   await ShoutoutsDao.updateShoutout({ ...shoutout, hidden: hide });
 };
+
+export const deleteShoutout = async (uuid: string, user: IdolMember): Promise<void> => {
+  const shoutout = await ShoutoutsDao.getShoutout(uuid);
+  if (!shoutout) {
+    throw new NotFoundError(`No shoutout with id '${uuid}' found.`);
+  }
+  const isLeadOrAdmin = await PermissionsManager.isLeadOrAdmin(user);
+  if (!isLeadOrAdmin || shoutout.giver.email !== user.email) {
+    throw new PermissionError(
+      `User with email: ${user.email} can't delete a shoutout from a different user!`
```

Comment:
It would be nice if this error message mentioned that it's also because you're not a lead/admin, since leads and admins can delete shoutouts from different users.

Reply:
Sounds good 👍